src/google_map.js

```diff
@@ -566,6 +566,9 @@ export default class GoogleMap extends Component {
 
         this._setLayers(this.props.layerTypes);
 
+        const versionMatch = maps.version.match(/^3\.(\d+)\./);
+        const mapsVersion = versionMatch ? Number(versionMatch[1]) : 99;
```

Comment:
Could we make this code more readable? What is `99`? or `versionMatch[1]`? (I understand ofc but its not so readable)

Reply:
Done.